css-cascade/source-order.css

```diff
@@ -0,0 +1,7 @@
+.first-class {
```

Comment:
you are missing one of the classes here.

Reply:
it's been done